competency-explorer/src/components/FilterBar/LevelFilters/LevelFilter/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback } from 'react';
+import cx from 'classnames';
+import { Level } from '../../../../models';
+import { useSelector, useDispatch } from 'react-redux';
+import { createFilterAction } from '../../../../reducers';
+import './styles.scss';
+import { selectLevelVisibility } from '../../../../selectors/visibility';
+
+export type LevelFilterProps = {
+    level: Level
+}
+
+export const LevelFilter = ({ level }: LevelFilterProps) => {
+    const isVisible = useSelector((s) => selectLevelVisibility(s, level))
+    const dispatch = useDispatch();
+    
+    const onClick = useCallback(() => {
+        dispatch(createFilterAction(level, 'TOGGLE_LEVEL'))
+    }, [isVisible])
```

Comment:
Did you mean to also set up [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks)? That would have caught that this `isVisible` isn't necessary.

Reply:
Did I mean to? No. Should I have? Yes. I'll add in the next PR :)